src/tests/ftest/aggregation/ec_aggregation_basic.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import time
+from ior_utils import run_ior
+from iot_test_base import IorTestBase
+from exception_utils import CommandFailure
+from job_manager_utils import get_job_manager
+
+
+class DaosECAggregationBasic(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description:
+
+       Run IOR with same file option to verify the
+       aggregation and 
+
+    :avocado: recursive
+    """
+    def get_free_space(self, storage_index):
+        """Get the pool information free space from the specified storage index.
+
+        Args:
+            storage_index (int): index of the pool free space to obtain
+
+        Returns:
+            int: pool free space for the specified storage index
+
+        """
+        self.pool.get_info()
+        return self.pool.info.pi_space.ps_space.s_free[storage_index]
+
+    def test_ec_aggregation(self, total_pools=1, total_containers_per_pool=1):
+        """Jira ID: DAOS-7326
+
+        Test Description:
+            This is a common method which run IOR on the specified pools,
+            container quanties and test aggregation.
+        """
+        # test params
+        test_oclass = self.params.get("oclass", '/run/aggregation_test/*')
+        total_runtime = self.params.get("total_runtime", 
+                                        '/run/aggregation_test/*')
+        pool_qty = total_pools
+        container_qty = total_containers_per_pool
+        start_time = 0
+        finish_time = 0
+        start_time = time.time()
+        while int(finish_time - start_time) < total_runtime:
+
+            job_manager = get_job_manager(self, "Mpirun", None, False, "mpich",
+                                          self.get_remaining_time())
+
+            # Create X pools
+            self.add_pool_qty(pool_qty, connect=False)
+            # Since the transfer size is 1M, the objects will be inserted
+            # directly into NVMe and hence storage_index = 1
+            storage_index = 1
+
+            initial_free_space = self.get_free_space(storage_index)
+
+            # Disable the aggregation
+            self.pool.set_property("reclaim", "disabled")
+
+            # Create Y containers per pool
+            for pool in self.pool:
+                self.add_container_qty(container_qty, pool)
+
+            # Run ior on each container sequentially
+            for container in self.container:
+                ior_log = "{}_{}_{}_ior.log".format(self.test_id,
+                                                    container.pool.uid,
+                                                    container.uuid)
+                try:
+                    self.ior_cmd.dfs_oclass.update(test_oclass)
+                    self.ior_cmd.dfs_dir_oclass.update(test_oclass)
+                    result = run_ior(self, job_manager, ior_log, self.hostlist_clients,
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'processes' in function call